tools/quantize/CMakeLists.txt

```diff
@@ -3,4 +3,5 @@ add_executable(${TARGET} quantize.cpp)
 install(TARGETS ${TARGET} RUNTIME)
 target_link_libraries(${TARGET} PRIVATE common llama ${CMAKE_THREAD_LIBS_INIT})
 target_include_directories(${TARGET} PRIVATE ../../common)
+target_include_directories(${TARGET} PRIVATE ../../src)
```

Comment:
This should be removed now that it doesn't include a private header.

Reply:
Good catch! Sorry for missing. Will push the update next.